packages/perseus-editor/src/article-editor.tsx

```diff
@@ -120,10 +133,13 @@ export default class ArticleEditor extends React.Component<Props, State> {
         };
     }
 
-    _sections(): ReadonlyArray<RendererProps> {
-        return Array.isArray(this.props.json)
-            ? this.props.json
-            : [this.props.json];
+    _sections(): PerseusRenderer[] {
+        const sections = Array.isArray(this.state.json)
+            ? this.state.json
+            : [this.state.json];
+        return sections.filter(
+            (section): section is PerseusRenderer => section !== null,
+        );
```

Comment:
Nit: Perhaps this could use nullish coalescence?
```
    _sections(): PerseusRenderer[] {
        return Array.isArray(this.state.json)
            ? (this.state.json ?? [])
            : [this.state.json];
    };
```